### PR TITLE
MoreUserTags: Add chat moderator tag

### PR DESCRIPTION
**Pull request by eepyfemboi**
added a trial mod tag because some servers only allow new mods to timeout members

### CODE REVIEW

**Comment by Vendicated** [comment]
i like the idea of this, but i don't think calling it a "Trial Moderator" is gonna be accurate for many users. Perhaps "Chat Moderator" would be more accurate?

cc @sunnniee

File: src/plugins/moreUserTags/index.tsx [hunk 1]
```diff
@@ -263,7 +269,16 @@ export default definePlugin({
...
+        // To prevent existing users from crashing on lines 379-380 due to the settings entry not existing (sry that its kinda sloppy)
+        if (settings.store.tagSettings && !settings.store.tagSettings['TRIAL_MODERATOR']) {
+            settings.store.tagSettings['TRIAL_MODERATOR'] = {
+                'text': 'T Mod', 
+                'showInChat': true, 
+                'showInNotChat': true
+            };
+            return;
+        }
+        else if (settings.store.tagSettings) return;
```
Comment (Vendicated) [inline src/plugins/moreUserTags/index.tsx:281]: i think the old migration has existed for long enough now, so feel free to just remove it

Comment (Vendicated) [reply src/plugins/moreUserTags/index.tsx:281]: actually i guess we still need the `settings.store.tagSettings = defaultSettings`, but the legacy migration is obsolete now

Comment (eepyfemboi) [reply src/plugins/moreUserTags/index.tsx:281]: Is there anything i should do about it? or..
Also i changed the name to "Chat Mod" like u mentioned earlier

Comment (Vendicated) [reply src/plugins/moreUserTags/index.tsx]: i did it for you!

**Review by Vendicated — APPROVED** [review]
looks good, thanks!

**Pull request merged**